.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,73 +1,64 @@
-name: "🐛 Bug report"
+name: "🕷️ Bug report"
 description: Report errors or unexpected behavior
-labels: 
+labels:
 - Issue-Bug
 - Needs-Triage
 body:
 - type: markdown
   attributes:
-    value: |
-      Please make sure to [search for existing issues](https://github.com/microsoft/PowerToys/issues) before filing a new one!
+    value: Please make sure to [search for existing issues](https://github.com/microsoft/PowerToys/issues) before filing a new one!
 - type: input
   attributes:
     label: Microsoft PowerToys version
-    placeholder: |
-      "0.53.0"
-    description: |
-      Hover over system tray icon or look at Settings
+    placeholder: 0.63.0
+    description: Hover over system tray icon or look at Settings
   validations:
     required: true
 
-- type: checkboxes
+- type: dropdown
   attributes:
     label: Running as admin
     description: Are you running PowerToys as Admin?
     options:
-      - label: "Yes"
+      - "Yes"
+      - "No"
 
 - type: dropdown
   attributes:
     label: Area(s) with issue?
-    description: What things had an issue?  Check all that apply.
+    description: What things had an issue? Check all that apply.
     multiple: true
     options:
       - General
       - Always on Top
       - Awake
       - ColorPicker
-      - Developer file preview
```

Comment:
why were all these removed

Reply:
In case it matters, I could rewrite;
"Thumbnail preview (Explorer)" ↔ "\<File> Explorer: Thumbnail Preview" and
"Preview Pane (Explorer)" ↔ "\<File> Explorer: Preview Pane"
The latter means they end up together in the list